.github/workflows/conda-python-build.yaml

```diff
@@ -51,44 +48,83 @@ permissions:
     statuses: none
 
 jobs:
-    build-test-upload:
-      strategy:
-        fail-fast: false
-      runs-on: linux-${{ inputs.CPU }}-gpu-${{ inputs.GPU }}-latest-1
+  compute-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      MATRIX: ${{ steps.compute-matrix.outputs.MATRIX }}
+    steps:
+      - name: Compute Build Matrix
+        id: compute-matrix
+        run: |
+          set -eo pipefail
+
+          # Please keep the matrices sorted in ascending order by the following:
+          #
+          #     [ARCH, PY_VER, CUDA_VER, LINUX_VER]
+          #
+          export MATRIX="
+          # amd64
+          - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04' }
```

Comment:
In the general case, the build matrix should be the cross-product of:
- amd64/arm64
- Every supported Python version (`3.10`, `3.11`, `3.12`, maybe `3.13`)
- Latest CUDA of each supported major version (`11.8.0`, `12.8.0` -- we don't have 12.8.1 images)
- `rockylinux8` (the oldest supported glibc among OSes that we have images for)

There are several differences that may apply to `numbast`:

I see this is filtering to just CUDA 12.8 and Python 3.11 in `pr.yaml`.
- If this package is pure (no Python or CUDA dependency) then you can drop the corresponding parts of the build matrix. There's no need to list a larger matrix here if you filter it later.
- Update to Python-latest (3.12 or 3.13)
- Does this project even support CUDA 11? Remove all traces if not.

Reply:
There is C++ to build, I think it was just a minor release behind with 3.9 still listed. I will take a pass and get to 3.10, 3.11 and 3.12, maybe punt on 3.13 for a follow up. I don't see evidence of CUDA 11 support, but I was planning on asking and then removing it all. Great feedback, I had been meaning to clean up the matrix.